test/issue-145.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const t = require('tap')
+const FindMyWay = require('../')
+
+const noop = function () {}
+
+t.test('issue-145', (t) => {
+  t.plan(4)
+
+  const findMyWay = FindMyWay({ ignoreTrailingSlash: true })
+
+  findMyWay.on('GET', '/a/b', noop)
+  findMyWay.on('GET', '/a/:pam/c', noop)
```

Comment:
Instead of using noop, can you please use named functions so that we can assert the right one matched?